src/components/mainLayout/curtainVk/curtainVk.tsx

```diff
@@ -0,0 +1,16 @@
+"use client";
+
+import * as VKID from "@vkid/sdk";
+
+import { ICurtainVk } from "../../../types/common/ComponentsProps";
+
+export const CurtainVK = (handleError: ICurtainVk) => {
+	const floatingOneTap = new VKID.FloatingOneTap();
+	floatingOneTap
+		.render({ appName: "freenance-app", scheme: VKID.Scheme.LIGHT, lang: VKID.Languages.RUS })
+		.on(VKID.WidgetEvents.ERROR, handleError);
+
```

Comment:
can you replace the a tag with a button tag in SignInForm and SignUpForm components in css classes signInFormContainer__auth_link and signUpFormContainer__auth_link adding the onClick attribute to call handleOpenAuthCurtain function?